containers/podman-desktop
# chore: enable svelte-check

### What does this PR do?

allows to check if code is valid
For now threshold is at error but it should be turned to hints
Fixed the last remaining errors

### Screenshot/screencast of this PR

N/A
### What issues does this PR fix or reference?

Fixes #461 

### How to test this PR?

Check menu: Pull Image and then click on 'add registry'
Adding registry checks should work as usual

Also the PR check should be green (svelte-check should work)


Change-Id: Id1af31a438775f38c8cf37ad75552fea9f21fe3e
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
